service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerTest.java

```diff
@@ -216,4 +204,104 @@ private CallbackRequest buildCallbackRequest() {
             .put("localAuthorityOrCafcass", LOCAL_AUTHORITY_NAME)
             .build();
     }
+
+    @Nested
+    class MidEvent {
+        private final byte[] pdf = {1, 2, 3, 4, 5};
+        private Document document;
+
+        @BeforeEach
+        void setUp() {
+            document = document();
+            DocmosisDocument docmosisDocument = new DocmosisDocument("order.pdf", pdf);
+
+            given(docmosisDocumentGeneratorService.generateDocmosisDocument(any(), any())).willReturn(docmosisDocument);
+            given(uploadDocumentService.uploadPDF(any(), any(), any(), any())).willReturn(document);
+        }
+
+        @Test
+        void midEventShouldGenerateOrderDocumentWhenOrderTypeIsBlankOrder() {
```

Comment:
I would maybe remove midEvent at the beginning of these tests. You get the context from the nested class :) what do you reckon? Won't reject but just a thought

Reply:
sure thing